src/components/NavBar/NavBar.tsx

```diff
@@ -119,6 +121,10 @@ const PressLogoType = styled(Typography)<{ display: boolean }>`
       opacity: 0;
       visibility: hidden;
     `}
+
+  ${(props) => lsdUtils.breakpoint(props.theme, 'xs', 'down')} {
+    display: none;
+  }
```

Comment:
Decided to hide the "Network State Press" text on the header for small screens. This has not yet been confirmed by Eddy, so it may change.